config/util/resource-locker/redis.json

```diff
@@ -8,11 +8,12 @@
       "locker": {
```

Comment:
Comment in this config can now change right? Since the locks are no longer stored in memory.

Reply:
Correct!